### DISCUSSION TITLE
What's "Parasitic Manager" ?

### DISCUSSION THREAD

**Discussion by xerta555** [LSPosed/LSPosed#1197] [Q&A] [answered]
Hi

I'm actually updating FR translation, but i don't understand what does mean "Parasitic Manager" in popup when i open the last BETA version of LSPosed Manager.

In the description, there is "system parasitization" that i don't understand.
![Screenshot_20210930-160440.jpg](https://user-images.githubusercontent.com/20888017/135470466-74774a7e-5b5c-4740-a65c-372a028f6339.jpg)

Some explanations are required for me to take hability to translate that on crowdin.
And i just been temporarily muted by the Rose bot today for no reason:
![image](https://user-images.githubusercontent.com/20888017/135470113-35756d9b-67a5-4e64-9f74-af59ec58478f.png)

Thanks to take a look in that.

Thanks.

**Comment by yujincheng08** [comment, marked as answer]
Parasitic manager means the manager that runs on the process of a system app.

**Reply by xerta555** [reply]
> 
> 
> Parasitic manager means the manager that runs on the process of a system app.

Ok, but what does this concretely bring in addition?

**Reply by yujincheng08** [reply]
app can no longer detect manager

**Reply by xerta555** [reply]
> 
> 
> app can no longer detect manager

Even other system apps couldn't detect the Manager?

**Reply by yujincheng08** [reply]
yup

**Reply by xerta555** [reply]
> 
> 
> yup

How is this possible?

**Reply by yujincheng08** [reply]
by parasitization

**Reply by xerta555** [reply]
> 
> 
> by parasitization

But what exactly does it involve?
I need to understand what it is in order to know what words i should use in the translation

**Reply by yujincheng08** [reply]
the manager is running on the process of a system app. the manager is not really installed on the system.

**Reply by xerta555** [reply]
> 
> 
> the manager is running on the process of a system app. the manager is not really installed on the system.

Ok i understand. However it's really the word "parasitization" where i get stuck, is it used to indicate that this mechanism is purely a parasite, or is it present to "confuse" detection?

**Reply by yujincheng08** [reply]
It means the manager is like a parasite living inside others

**Reply by xerta555** [reply]
> 
> 
> It means the manager is like a parasite living inside others


So you're telling me that it acts like a virus that infiltrates system apps?
